acl.go

```diff
@@ -48,6 +49,10 @@ func backendReferenceByAcl(frontend Frontend, backends []Backend) error {
 		for _, a := range ub.Condition.Any {
 			// TODO support or/and conditions
 			for _, s := range a {
+				//TODO Handle ! correctly instead of ignoring it
+				if strings.HasPrefix(s, "!") {
+					s=strings.TrimLeft(s,"!")
```

Comment:
Could you apply `gofmt` to follow Go's coding style?

Reply:
Sure. Thanks for the suggestion

Reply:
👍